src/redux/books/books.js

```diff
@@ -1,30 +1,17 @@
-import { createAsyncThunk } from '@reduxjs/toolkit';
-
 // Actions
 const ADD = 'bookstore-react/booksReducer/ADD';
 const REMOVE = 'bookstore-react/booksReducer/REMOVE';
-const GET = 'bookstore-react/booksReducer/GET';
-
-// API URL
-const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/xKgONp1vYVZJzmgq5zMK/books';
 
 // Reducer
 export default function booksReducer(state = [], action) {
   switch (action.type) {
-    case `${ADD}/fulfilled`:
-      return state.concat(action.meta.arg);
-    case `${REMOVE}/fulfilled`:
-      return state.filter((book) => book.item_id !== action.meta.arg);
-    case `${GET}/fulfilled`:
-      return Object.keys(action.payload).map((key) => {
-        const { title, author, category } = action.payload[key][0];
-        return {
-          item_id: key,
-          title,
-          author,
-          category,
-        };
-      });
+    case ADD:
+      return [
+        ...state,
```

Comment:
- [ ] According to the project you should  Fetch your books from Bookstore API on when the component has rendered with` useEffect` . Please fix it 👍